chrome-extension/background/utils/executeActionsScript.js

```diff
@@ -6,6 +6,16 @@ const handleType = (element, value) => {
   element.value = value;
 };
 
+const handleSelect = (element, value) => {
+  for (let i = 0; i < element.options.length; i++) {
+    if (element.options[i].value == value) {
```

Comment:
lets make this case insensitive, so "march" and "March" would match the same validation

Reply:
use `innerText` instead of `value`
```suggestion
    if (element.options[i].innerText == value) {
```